src/wayland/mod.rs

```diff
@@ -51,13 +51,15 @@ pub mod buffer;
 pub mod compositor;
 pub mod data_device;
 pub mod dmabuf;
+pub mod input_method_manager;
```

Comment:
Also a little nitpicky, but generally the modules are named after the protocols.
And these protocols are named `input_method` and `text_input` not `*_manager`, that is just the internal type.
So if you don't mind please rename these.

Reply:
Done :)